app/scripts/views/settings.js

```diff
@@ -206,6 +206,7 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
         })
         .fin(function () {
           self.logViewEvent('signout.success');
+          self.relier.unset('uid');
```

Comment:
You are digging up a bunch of dirt! Do you think we should also clear the [email field](https://github.com/mozilla/fxa-content-server/blob/2a51116d1450b3f20c570328c512f5e78b9d01f5/app/scripts/models/reliers/relier.js#L38) and add an entry in `defaults` for `uid` in the relier while we are at it?